packages/runc/Cargo.toml

```diff
@@ -16,6 +16,11 @@ url = "https://github.com/opencontainers/runc/releases/download/v1.1.14/runc.tar
 path = "runc-v1.1.14.tar.xz"
 sha512 = "71c21c2cff82402d937163e73805dc4f73eca948d5a05c2fe2aa0b1161438adf538caad2e3b98d9e0b786d7f04c01971a494a1fc3e24fe94fb76e64bf9453ad9"
 
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/opencontainers/runc/releases/download/v1.1.14/runc.tar.xz.asc"
```

Comment:
We just merged runc-1.1.15 :) 
https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/323